docs/source/developer_guide/coding_standards.rst

```diff
@@ -88,22 +84,21 @@ checks on the files you changed whenever you make a new commit.
 You can find our pre-commit configuration in
 `.pre-commit-config.yaml <https://github.com/sktime/sktime/blob/main/.pre-commit-config.yaml>`_.
 Additional configurations can be found in
-`setup.cfg <https://github.com/sktime/sktime/blob/main/setup.cfg>`_.
+`pyproject.toml <https://github.com/sktime/sktime/blob/main/pyproject.toml>`_.
 
 .. note::
-   If you want to exclude some line of code from being checked, you can add a ``# noqa`` (no quality assurance) comment at the end of that line.
+   If you want to exclude some line of code from being checked, you can add a ``# noqa: rule`` (no quality assurance) comment at the end of that line.
+   The set of rules that can be excluded can be found here `ruff rules <https://docs.astral.sh/ruff/rules>`_.
 
 Integrating with your local developer IDE
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Local developer IDEs will usually integrate with common code quality checks, but need setting them up in IDE specific ways.
 
-For Visual Studio Code, ``black``, ``flake8``, ``isort`` and/or ``numpydoc`` will need to be activated individually in the preferences
-(e.g., search for ``black`` and check the box). The packages ``black`` etc will need to be installed in the python environment used by the IDE,
+For Visual Studio Code, ``ruff`` and/or ``numpydoc`` will need to be activated individually in the preferences you can install ``ruff`` vscode extension from the marketplace.
```

Comment:
Can you add link to the extension please?

Also, I believe the extension do not respect `pyproject.toml` directly and separate configuration (possibly per workspace) is required. Can you please verify? If so, we may add instructions on how to do so.

Reply:
It fully conforms to the `ruff` configurations in any of `ruff.toml` or `pyproject.toml`

Reply:
I am using [this](https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff) for office works, and it fails to do that automatically in a workspace. Not sure if it depends on operating system or version etc.

Not important to add those instructions, we can skip.

Reply:
They mentioned this link you shared, I tried it and it works from my end. I think it might be an issue with your system